src/test/ui/let-else/issue-102317.rs

```diff
@@ -0,0 +1,20 @@
+// issue #102317
+// run-pass
```

Comment:
Should be check-pass or build-pass I think

Reply:
I think I will choose
```suggestion
// build-pass
```